docs/guides/extensions/interfaces-radio-selector-icons.md

```diff
@@ -157,6 +157,12 @@ select-icon, which provides a searchable dropdown of the icon library.
 
 ## Work With Images
 
+::: tip
+
+[DEPRECATED] Use session cookies instead.
```

Comment:
Could we add a link? I guess to the SDK authentication guide

Reply:
Oh right, to the API auth page 👍 ❤️

Reply:
Perhaps a link to the break change note is most appropriate 🤔 As whether this is enabled by default or not will depend on the version you're running

Reply:
Great idea 🚀